docs/release_notes.adoc

```diff
@@ -207,6 +207,8 @@ Spy(constructorArgs: [null as String, (Pattern) null])
 def "I'll run everywhere but on Windows"() { ... }
 ----
 
+- Execution with the unsupported Groovy version can enforced with `-Dspock.iKnowWhatImDoing.disableGroovyVersionCheck=true`
```

Comment:
> Execution with the unsupported Groovy version can enforced with

I would rephrase that to

`Execution with an unsupported Groovy version can allowed with`

Reply:
Chagned.